src/components/sellers/sections/FeaturedTestimonial.js

```
@@ -2,56 +2,59 @@ import React from "react";
 import "../../../styles/components/sellers/sections/FeaturedTestimonial.css";
 import sellerImage from "../../../assets/logos/seller_closing.jpg";
 
-const FeaturedTestimonial = ({ backgroundColor = "#fff" }) => {
+const FeaturedTestimonial = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)